### PR TITLE
refactor: unify int type

### PR DESCRIPTION
**Pull request by wfxr**

### CODE REVIEW

File: internal/database/online/redis/util.go [hunk 1]
```diff
@@ -18,21 +18,10 @@ func SerializeByTag(i interface{}, typeTag string) (s string, err error) {
...
 		return i.(string), nil
-
-	case types.INT8:
-		return strconv.FormatInt(int64(i.(int8)), SeralizeIntBase), nil
-	case types.INT16:
-		return strconv.FormatInt(int64(i.(int16)), SeralizeIntBase), nil
-	case types.INT32:
-		return strconv.FormatInt(int64(i.(int32)), SeralizeIntBase), nil
-	case types.INT64:
+	case types.INT8, types.INT16, types.INT32, types.INT64:
```
Comment (YikSanChan) [inline internal/database/online/redis/util.go:21]: can you add comments?

Comment (wfxr) [reply internal/database/online/redis/util.go:21]: I will remove it in next pr.

**Review by YikSanChan — APPROVED** [review]
lgtm

File: internal/database/online/redis/util.go (hunk 1 above)
Comment (wfxr) [reply internal/database/online/redis/util.go]: @YikSanChan fixed

**Review by lianxmfor — APPROVED** [review]
lgtm

**Pull request merged**